shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -81,6 +84,9 @@
     private final BroadcastReceiver mDiscoveryReceiver;
     private final List<ActivityLifecycleListener> mActivityLifecycleListeners;
     private long mNativePlatformView;
+    private boolean isSoftware = false; //using the software renderer or not
```

Comment:
`mIsUsingSoftwareRendering` follows the convention regarding member variables and is more descriptive.